ods_ci/tests/Resources/Files/llm/vllm/vllm_servingruntime.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: serving.kserve.io/v1alpha1
+kind: ServingRuntime
+metadata: 
+  name: kserve-vllm
+  namespace: vllm-gpt2
+spec: 
+  annotations: 
+    sidecar.istio.io/inject: "true"
+    sidecar.istio.io/rewriteAppHTTPProbers: "true"
+    serving.knative.openshift.io/enablePassthrough: "true"
+    opendatahub.io/dashboard: "true"
+    openshift.io/display-name: "vLLLM Openai entry point"
+    serving.kserve.io/enable-prometheus-scraping: "false"
+    serving.kserve.io/enable-metric-aggregation: "true"
```

Comment:
```suggestion
```

Reply:
@VedantMahabaleshwarkar what value is expected for this annotation? `serving.kserve.io/enable-metric-aggregation`

Reply:
we are not using `serving.kserve.io` annotations for prometheus. So the annotation should not exist. 
We need `prometheus.io/port` + `prometheus.io/path`

Reply:
FWIW I followed the information found in https://github.com/opendatahub-io/odh-model-controller/pull/200 and RHOAIENG-6264; I can remove the annotations however